browser/src/ShortTandemRepeatPage/ShortTandemRepeatAttributes.tsx

```diff
@@ -4,60 +4,59 @@ import AttributeList, { AttributeListItem } from '../AttributeList'
 import InlineList from '../InlineList'
 import Link from '../Link'
 
-import { ShortTandemRepeat } from './ShortTandemRepeatPage'
+import { ShortTandemRepeat, RepeatUnitClassification } from './ShortTandemRepeatPage'
 
 type ShortTandemRepeatRepeatUnitsProps = {
   shortTandemRepeat: ShortTandemRepeat
 }
 
 const ShortTandemRepeatRepeatUnits = ({ shortTandemRepeat }: ShortTandemRepeatRepeatUnitsProps) => {
-  const repeatUnitsByClassification: Record<string, string[]> = {}
+  const repeatUnitsByClassification: Partial<Record<RepeatUnitClassification, string[]>> = {}
   shortTandemRepeat.repeat_units.forEach((repeatUnit) => {
     if (repeatUnitsByClassification[repeatUnit.classification] === undefined) {
       repeatUnitsByClassification[repeatUnit.classification] = []
     }
-    repeatUnitsByClassification[repeatUnit.classification].push(repeatUnit.repeat_unit)
+    repeatUnitsByClassification[repeatUnit.classification]!.push(repeatUnit.repeat_unit)
   })
 
   if (
-    !(repeatUnitsByClassification as any).pathogenic &&
-    !(repeatUnitsByClassification as any).benign
+    !repeatUnitsByClassification.pathogenic &&
+    !repeatUnitsByClassification.benign &&
+    repeatUnitsByClassification.unknown
   ) {
     return (
       <AttributeListItem
-        label={`Repeat unit${(repeatUnitsByClassification as any).unknown.length > 1 ? 's' : ''}`}
+        label={`Repeat unit${repeatUnitsByClassification.unknown.length > 1 ? 's' : ''}`}
       >
         <InlineList
-          items={(repeatUnitsByClassification as any).unknown.map((repeatUnit: string) => (
+          items={repeatUnitsByClassification.unknown.map((repeatUnit: string) => (
             <span>
               {repeatUnit === shortTandemRepeat.reference_repeat_unit &&
               shortTandemRepeat.repeat_units.length > 1
                 ? `${repeatUnit} (reference)`
                 : repeatUnit}
             </span>
           ))}
-          label={`Repeat unit${(repeatUnitsByClassification as any).unknown.length > 1 ? 's' : ''}`}
+          label={`Repeat unit${repeatUnitsByClassification.unknown.length > 1 ? 's' : ''}`}
         />
       </AttributeListItem>
     )
   }
 
   if (
-    (repeatUnitsByClassification as any).pathogenic &&
-    (repeatUnitsByClassification as any).pathogenic.length === 1 &&
-    !(repeatUnitsByClassification as any).benign &&
-    !(repeatUnitsByClassification as any).unknown
+    repeatUnitsByClassification.pathogenic &&
+    repeatUnitsByClassification.pathogenic.length === 1 &&
+    repeatUnitsByClassification.benign &&
+    repeatUnitsByClassification.unknown
```

Comment:
It looks like the `!` may have been mistakenly dropped from these last two conditions:

before:

```
(repeatUnitsByClassification as any).pathogenic &&
(repeatUnitsByClassification as any).pathogenic.length === 1 &&
!(repeatUnitsByClassification as any).benign &&
!(repeatUnitsByClassification as any).unknown
```

after:

```
repeatUnitsByClassification.pathogenic &&
repeatUnitsByClassification.pathogenic.length === 1 &&
repeatUnitsByClassification.benign &&
repeatUnitsByClassification.unknown
```

Just wanted to flag this in case it was unintentional!